ftw/solr/browser/resources/ftwtheming.scss

```diff
@@ -179,7 +179,7 @@ $dropdown-link-color-hover: $link-hover-color !default;
 
         @include boxshadow(0.5em 1em 2em -1em $dropdown-border-color);
 
-        @include borderradius($border-radius);
+        @include borderradius($border-radius-primary);
```

Comment:
`borderradius` mixin is not necessary. According to http://caniuse.com/#search=border-radius we need no vender prefixes for `border-radius`.